src/Server.php

```diff
@@ -77,7 +101,7 @@ class Server extends AbstractServer
      * @return Server
      * @throws Exception\InvalidArgumentException If function invalid or not callable.
      */
-    public function addFunction($function, $namespace = '')
+    public function addFunction($function, $namespace = '') : Server
```

Comment:
Missing param types here.

Is it because we extend abstract?
Maybe in that case will be worth also update laminas-server first?

Reply:
Correct, method signature is defined by `Server` interface of `laminas/laminas-server`.
I am willing to help upgrade that package, too. Since I will be on a leave and unavailable in February and March, I would like to complete this package first - if possible.